.changeset/good-pants-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-data": minor
+---
+
+Add `purgeCaches` export for purging all caches with one call
```

Comment:
I did each chunk of API or renaming as a commit and did a changeset for each one to keep track. Put up as a single PR because it didn't seem worth splitting - I can if necessary though.